omnibus/config/software/python2.rb

```diff
@@ -35,6 +35,8 @@
   env = {
     "CFLAGS" => "-I#{install_dir}/embedded/include -O2 -g -pipe -fPIC",
     "LDFLAGS" => "-Wl,-rpath,#{install_dir}/embedded/lib -L#{install_dir}/embedded/lib",
+    "PKG_CONFIG" => "#{install_dir}/embedded/bin/pkg-config",
```

Comment:
🍰

Reply:
nit: this makes the software def rely on the `pkg-config` binary so `pkg-config` should be added as a dependency of the software def